src/main.js

```diff
@@ -1,34 +1,84 @@
 
-// Create variables targetting the relevant DOM elements here 👇
```

Comment:
Because these two pieces of code are different, it's a bit tricky to give live numbers for how to reorganize. Instead, perhaps we can decided an order that outlines where each type of code is placed, from top to bottom. Here's my suggestion.

Reply:
1. Global variables: savedCovers, cover piece variables (coverImage, coverTitle, etc), global view variables (homeView, formView,etc), global control variables (buttons, etc)
2. Event listeners: anything listening for loads, clicks, etc
3. Event handler functions: switch to formView, etc
4. Other functions: getRandomIndex, createRandomCover, updateCover